projects/plugins/starter-plugin/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );
+
+module.exports = {
+	...baseConfig,
+	roots: [ '<rootDir>/src' ],
```

Comment:
Hmm, it's copy pasted from another project. 

> this breaks the ability to have tests under tests/ 

I guess this is not a problem, since the pattern I see with react jest tests is to keep them close to the components. Do you think it's worth updating it to pick up tests from `tests/`?

Reply:
Maybe it's just me that does it, I don't see JS tests under `tests/` anywhere other than in code I wrote. I'm ok with leaving this.